fs-one-concepts.md

```diff
@@ -4,18 +4,80 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
 
+My answer:
+ - Enzyme is a testing library for React that adds methods for testing functionality.
```

Comment:
You found some great information in your research. The first 2 points are key to what Enzyme provides for us.